packages/now-client/src/deploy.ts

```diff
@@ -231,3 +233,5 @@ export default async function* deploy(
     }
   }
 }
+
```

Comment:
Weird spacing? Maybe run `prettier` on this file

Reply:
Done ✅